README.md

```diff
@@ -52,34 +52,35 @@ The `tests` directory represents the testing environment for ensuring native fun
 Install Testing Environment Dependencies
 
 ```bash
-$ npm install
+$ yarn install
 ```
 
 Install Local jschart Node Module
 
 ```bash
-$ npm install --save ../jschart/
+$ cd jschart  
+$ yarn link
+$ cd ../tests 
+$ yarn link "jschart"
 ```
 
-This will create a reference to the `jschart/` directory under dependencies in `package.json`:
+Verify Local jschart Node Module Installation
 
-```
-"dependencies": {
-  "jschart": "file:jschart"
-}
+```bash
+$ readlink node_modules/jschart
```

Comment:
Do we want to suggest the use of the -e argument to readlink?  Without it I get this:

```
$ readlink node_modules/jschart
../../../../../.config/yarn/link/jschart
```

Which is just pointing to yet another symbolic link.  With -e it I get a complete resolution to the actual file:

```
$ readlink -e node_modules/jschart
/home/krister/data/code/jschart/jschart
```

Reply:
Thanks for catching that @k-rister, just pushed updated changes.